console/console-init/ui/src/modules/iot-device/DeviceListPage.tsx

```diff
@@ -44,8 +49,38 @@ export default function DeviceListPage() {
     getInitialFilter()
   );
 
+  const [isAlertVisible, setIsAlertVisible] = useState<boolean>(false);
+  const [alertVariant, setAlertVariant] = useState<AlertVariant>();
+  const [alertTitle, setAlertTitle] = useState<string>("");
+  const [alertDescription, setAlertDescription] = useState<string>("");
+
   const { dispatch } = useStoreContext();
 
+  useEffect(() => {
+    if (totalDevices && totalDevices < MAX_DEVICES) setIsAlertVisible(false);
```

Comment:
please move this whole code into function and call that function in useEffect hook.

Reply:
Done